api/PclusterApiHandler.py

```diff
@@ -271,6 +271,50 @@ def submit_job():
     return resp if type(resp) == tuple else {"success": "true"}
 
 
+def translate_job(body, user):
```

Comment:
suggestion: I found the whole function (naming in particular) a little bit convoluted :)

What do you think of a refactoring? Something like:

```
def translate_job(body, user):
    job_properties = {
        "job-name": "name",
        "chdir": "current_working_directory"
    }
    
    translated_job = {
        "job": {
            "environment": {
                "PATH":"/bin:/usr/bin/:/usr/local/bin/"
            }
        }
    }

    body['chdir'] = f"/home/{user}"    
    for key in body:
        if key == 'command':
            translated_job['script'] = body['command']
        else:
            translated_job['job'][job_properties.get(key, key)] = body[key]

    return translated_job
```

Reply:
You're right, it's isn't very clear -  did my best to clean it up